Brokerages/Oanda/OandaRestApiV1.cs

```diff
@@ -64,6 +64,15 @@ public OandaRestApiV1(OandaSymbolMapper symbolMapper, IOrderProvider orderProvid
         {
         }
 
+        /// <summary>
+        /// Gets the account base currency
+        /// </summary>
+        public override string GetAccountBaseCurrency()
```

Comment:
Does OANDA still support V1 accounts?

Reply:
The V1 API was deprecated two years ago:
https://developer.oanda.com/rest-live/introduction/

Existing V1 accounts were migrated to V20 and users received this notice:
`
On 8 December 2017 we will be migrating your legacy trade account(s) to a new V20 account(s). This will enable OANDA to ensure that your account(s) meet the anticipated regulatory requirements at the beginning of 2018 so you can continue to trade. On this date the funds in your legacy account(s) and any open positions will be transferred to your new V20 account(s). 
`

I think we can safely remove the Oanda V1 implementation from `OandaBrokerage` in a separate PR.